### DISCUSSION TITLE
'"react-unity-webgl"' has no exported member named 'UnityContext'. Did you mean 'UnityContent'?

### DISCUSSION THREAD

**Discussion by renatomarquesteles** [jeffreylanters/react-unity-webgl#350] [Help Wanted!] [answered]
My Unity project is version 2019.4.20f1
then I installed "react-unity-webgl": "7.1.13" as suggested in the README
but in the import I'm getting this error:
'"react-unity-webgl"' has no exported member named 'UnityContext'. Did you mean 'UnityContent'?

![image](https://user-images.githubusercontent.com/37571156/158686538-828a7db1-655e-4fe5-9d15-0b4f0f66da9e.png)

"react": "17.0.2"

**Comment by jeffreylanters** [comment, marked as answer]
Hi. You should head over to the 7.x documentation.

**Reply by renatomarquesteles** [reply]
> Hi. You should head over to the 7.x documentation.

sorry, how do I access it?

**Reply by renatomarquesteles** [reply]
> > Hi. You should head over to the 7.x documentation.
> 
> sorry, how do I access it?

got it. Thank you!

**Reply by jeffreylanters** [reply]
Awesome! Best of luck with your project!

**Reply by MomasVII** [reply]
Where can we find the 7.x documentation?
